core/controllers/acl_decorators.py

```diff
@@ -2753,6 +2753,73 @@ def test_can_accept_suggestion(
     return generate_decorator_for_handler
 
 
+def can_update_suggestions(handler):
+    """Function that generates a decorator for a given handler.
+
+    Args:
+        handler: function. The function to be decorated.
+
+    Returns:
+        function. The newly decorated function that has common checks and
+        permissions specified by passed in decorator.
```

Comment:
This is too general—can you specify what permissions are actually being checked?

Reply:
Done